src/components/DocumentationTopic/ContentTable.vue

```diff
@@ -43,12 +43,6 @@ export default {
 <style scoped lang="scss">
 @import 'docc-render/styles/_core.scss';
 
-.contenttable {
-  @include inTargetIde {
-    background: var(--color-content-table-content-color);
```

Comment:
We may want to consider dropping the `--color-content-table-content-color` variable entirely as well if this is the only place it was used.